xml/html_title.xml

```diff
@@ -3247,4 +3391,332 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:moinmo:moinmoin:-"/>
   </fingerprint>
 
-</fingerprints>
+  <fingerprint pattern="^(\w+) - BlackBox Management Console$">
+    <description>BlackBox Management Console</description>
+    <example host.name="ABC123">ABC123 - BlackBox Management Console</example>
+    <param pos="0" name="hw.vendor" value="BlackBox"/>
+    <param pos="0" name="hw.device" value="Device Server"/>
+    <param pos="0" name="hw.product" value="BlackBox"/>
+    <param pos="1" name="host.name"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Shelly Color Bulb$">
+    <description>Shelly Smart Bulb</description>
+    <example>Shelly Color Bulb</example>
+    <param pos="0" name="os.vendor" value="Cesanta"/>
+    <param pos="0" name="os.product" value="Mongoose"/>
+    <param pos="0" name="hw.vendor" value="Shelly"/>
+    <param pos="0" name="hw.device" value="Light Bulb"/>
+    <param pos="0" name="hw.product" value="Color Bulb"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Shelly (Plug|Dimmer(?: 2)?|Switch)$">
+    <description>Shelly Plug/Dimmer/Switch Devices</description>
+    <example hw.product="Plug">Shelly Plug</example>
+    <example hw.product="Dimmer">Shelly Dimmer</example>
+    <example hw.product="Dimmer 2">Shelly Dimmer 2</example>
+    <example hw.product="Switch">Shelly Switch</example>
+    <param pos="0" name="os.vendor" value="Cesanta"/>
+    <param pos="0" name="os.product" value="Mongoose"/>
```

Comment:
Here and elsewhere, this `os.product` should be Mongoose OS or we should add a CPE remap

`o:cesanta:mongoose_os`